files/image_config/interfaces/interfaces-config.sh

```diff
@@ -18,8 +18,14 @@ else
     echo "{ \"ZTP_DHCP_DISABLED\" : \"true\" }" > /tmp/ztp_input.json
 fi
 
-# Create /e/n/i file for existing and active interfaces
-sonic-cfggen -d -j /tmp/ztp_input.json -t /usr/share/sonic/templates/interfaces.j2 > /etc/network/interfaces
+# Create /e/n/i file for existing and active interfaces, dhcp6 sytcl.conf and dhclient.conf
+CFGGEN_PARAMS=" \
+    -d -j /tmp/ztp_input.json \
+    -t /usr/share/sonic/templates/interfaces.j2,/etc/network/interfaces \
+    -t /usr/share/sonic/templates/90-dhcp6-systcl.conf.j2,/etc/sysctl.d/90-dhcp6-systcl.conf \
```

Comment:
looks like a typo in systcl (sysctl) but its from the old code.. :)

Reply:
It is a type. This will a bigger change

Reply:
Just searched and it is limited to this container (script, and copy constructs)